src/software/ai/passing/cost_function.cpp

```diff
@@ -173,72 +142,64 @@ double calculateInterceptRisk(const Team& enemy_team, const Pass& pass,
     }
     std::vector<double> enemy_intercept_risks(enemy_robots.size());
     std::transform(enemy_robots.begin(), enemy_robots.end(),
-                   enemy_intercept_risks.begin(), [&](Robot robot) {
-                       return calculateInterceptRisk(robot, pass, enemy_reaction_time);
+                   enemy_intercept_risks.begin(), [&](const Robot& robot) {
+                       return calculateInterceptRisk(robot, pass, passing_config);
                    });
     return *std::max_element(enemy_intercept_risks.begin(), enemy_intercept_risks.end());
 }
 
 double calculateInterceptRisk(const Robot& enemy_robot, const Pass& pass,
-                              const Duration& enemy_reaction_time)
+                              const TbotsProto::PassingConfig& passing_config)
```

Comment:
Enemy interception rating was updated to not use Sigmoid. Using Sigmoid caused the ratings to always be >0, even when we're directly shooting into an enemy robot. The updated implementation uses the delta in ball travel time and the estimated enemy interception time to create a rating. As a result, we could have ratings of 0 when we're passing through an enemy, and ratings of 1 when we're passing far from all enemies.